src/MusicStore/Models/SampleData.cs

```diff
@@ -111,6 +111,20 @@ public static class SampleData
                 //await userManager.AddToRoleAsync(user, adminRole);
                 await userManager.AddClaimAsync(user, new Claim("ManageStore", "Allowed"));
             }
+
+            var envPerfLab = configuration.Get<string>("PERF_LAB");
+            if (envPerfLab == "true")
+            {
+                for (int i = 0; i < 100; ++i)
+                {
+                    var email = string.Format("User{0:D3}@sample.com", i);
```

Comment:
It's updated https://github.com/aspnet/MusicStore/commit/63f6f42e7e189d2f60a74a09334137769a2548ec